programs/zstd.1.md

```diff
@@ -363,6 +363,11 @@ Compression of small files similar to the sample set will be greatly improved.
 
     `zstd --train-legacy=selectivity=8 FILEs`
 
+* `--show-default-cparams`:
```

Comment:
It's positioned in the "dictionary builder" category, which is probably not the right category for this command.

Also, could you explain to me the last sentence ?

Reply:
Whoops, I meant to put it under options. I'm not sure which category it goes under in there. Integer suffixes and special values, Operation mode, Restricted usage of Environment Variables don't seem apt. Maybe I'll create a new section called misc. arguments?

Reply:
It's a kind of request for some supplemental information display, 
so it could be in the same category as `-v` / `--verbose`.